src/__deprecated__/components/simple-color-picker/simple-color-picker.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import ColorOption from './color-option';
-import tagComponent from '../../utils/helpers/tags';
+import tagComponent from '../../../utils/helpers/tags/tags';
```

Comment:
I think this can be simplified to:
```suggestion
import tagComponent from '../../../utils/helpers/tags';
```